docs/quickstarts/insights-inventory-workspace/insights-inventory-workspace.yaml

```diff
@@ -96,36 +94,52 @@ spec:
     - title: Remove systems from a workspace
       description: |- 
 
-        **To remove systems from a workspace from the workspace's page:**
+        To remove systems from a workspace:
 
-        1. Navigate to Inventory > **Workspaces** using the left navigation.
-        1. Select systems you want to remove from a workspace.
-            
-            - **To remove individual systems from a workspace**, click on <i class="fas fa-ellipsis-v"></i> in the row level, then click **Remove from workspace**. 
+        1. Go to **RHEL** > **Inventory** to see your list of systems and the workspaces they belong to. 
+
+        2. To remove one system from a workspace:
+
+            a. Click the <i class="fas fa-ellipsis-v"></i> (options icon) next to the system, then click **Remove from workspace**. 
```

Comment:
Wouldn't this be the other way around so that the sentence is complete without the content of the brackets?
Click options icon (<i class="fas fa-ellipsis-v"></i>) next to ...